src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -262,6 +258,86 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         }
     }
 
+    /**
+    * @description Evaluate the state of the RD2 enablement
+    *   - 0 => RD2 Not enabled
+    *   - 1 => RD2 enabled, but MetaDeploy step was skipped
+    *   - 2 => RD2 enabled, metaDeploy is completed, NextPaymentDate rollup still enabled
+    *   - 3 => RD2 fully Enabled
+    */
+    private void handleRD2EnablementState() {
+        Integer rd2EnablementStateIndicator;
+
+        if (!RD2_EnablementService.isRecurringDonations2Enabled) {
+            rd2EnablementStateIndicator = 0;
+
+        } else if (isMetaDeployNotCompleted()) {
+            rd2EnablementStateIndicator = 1;
+
+        } else if (!isNextPaymentDateRollupDisabled()) {
+            rd2EnablementStateIndicator = 2;
+
+        } else {
+            rd2EnablementStateIndicator = 3;
```

Comment:
@andrewyu-salesforce I was not really thinking ahead when I came up with this number scheme. For example, 3.179 will include a new custom UI with New/Edit overrides in the unpackaged metadata (and likely other small changes). It would be nice if we could eventually detect that the new UI isn't being used and include that in our results. 

Anyway, what you think of changing "3" to "10" here and the unit test to represent that RD2 is fully enabled? That gives us a little more wiggle room to add in additional metrics over time, _if needed_.

Reply:
Changing to 10 make sense if we are going to track more step in the future. I will do the update.

Reply:
Thank you. I updated the WI description to match.